blogposts/2022/sourcegraph-non-devs.md

```diff
@@ -0,0 +1,207 @@
+---
+title: 'Sourcegraph for non-devs'
```

Comment:
Could we tweak the title to be more explicit about what people will get out of the post? There are some great examples in the post about how Sourcegraph helped so it'd be great to lead with something like that